tests/rptest/services/redpanda.py

```diff
@@ -368,6 +401,18 @@ class RedpandaService(Service):
             "path": STDOUT_STDERR_CAPTURE,
             "collect_default": True
         },
+        "redpanda_tls_server_key_file": {
```

Comment:
I guess this is for debugging something going wrong with the TLS cert generation -- do you think we need to leave it in permanently?

Reply:
I wasn't using it for debugging, but for some vague idea that it'd be useful for future debugging. will remove